dts/msm8916-samsung-r06.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0xCE08FF01 6>;
+
+	gt58ltebmc {
+		model = "Samsung Galaxy Tab A 8.0";
+        compatible = "samsung,gt58ltebmc", "qcom,msm8916-mtp", "qcom,msm8916", "qcom,mtp", "lk2nd,device";
+		lk2nd,match-bootloader = "T35*";
```

Comment:
Please make this as strict as possible i.e. `T357W*` if that works. lk2nd displays the full bootloader string in the fastboot menu, so you can check it there.